### DISCUSSION TITLE
[Question][Hw4] git commands are not working after Killing the test

### DISCUSSION THREAD

**Discussion by Gimongjin** [prosyslab-classroom/cs424-program-reasoning#338] [Q&A]
During the implementation, I ran the test for `example4.sl`.
Then immediately the SSH connection went bad and disconnected.
When I restored the connection, the result was written as `Killed`. 
I think this is due to my poor equivalence reduction, but the problem is after such situation, git commands are not working at all.
`git clone` and `git push` commands are not working anymore until now.
Is there any problem in my virtual machine account?

Edit: only the git commands for my homework 4 repository is not working. cloning or committing other repository still works.

**Comment by Hhro** [comment]
I cannot see any wrong with your VM account.
Have you tried cloning the homework4 repository into another directory and pushing it?

**Comment by Gimongjin** [comment]
After waiting couple hours it started to work well.
I think it was the contemporary problem. If you are running out of time then it will be better to commit and push your answer first and then run some complicated test cases.
